test/integration/server-selection/readpreference.test.js

```diff
@@ -1,10 +1,10 @@
 'use strict';
 
-const test = require('./shared').assert;
-const setupDatabase = require('./shared').setupDatabase;
-const withMonitoredClient = require('./shared').withMonitoredClient;
-const { ReadPreference, Topology } = require('../../src');
-const { withClient } = require('./shared');
+const test = require('../../functional/shared').assert;
+const setupDatabase = require('../../functional/shared').setupDatabase;
+const withMonitoredClient = require('../../functional/shared').withMonitoredClient;
```

Comment:
```suggestion
const { assert: test } = require('../../functional/shared');
const { setupDatabase } = require('../../functional/shared');
const { withMonitoredClient } = require('../../functional/shared');
```

Reply:
just to add on to the suggestion - once they're in this form, they can all be on one line since they're imported from the same file